ui/admin/app/components/form/host-set/aws/index.js

```diff
@@ -0,0 +1,17 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class FormHostSetAwsComponent extends Component {
+  // =actions
+  @tracked showSyncIntervalSeconds = true;
+
+  @action
+  toggleSyncIntervalSeconds(model) {
```

Comment:
Assume I might be missing context, so if I ask something too obvious let me know.

We are declaring `toggleSyncIntervalSeconds` within `components/form/host-set/aws/` wich is ok.
But we are using it in `form/host-set/aws` and `form/host-set/azure` templates. Shouldn't this be created in `form/host-set/index.js`. 

I see this functions beeing shared between these child templates, but being declared on a child index. If is shared between childs, I would preffer to have it declared at parent level.

Reply:
good point! I'll move this up to the parent level